indexing-hadoop/src/main/java/io/druid/indexer/InputRowSerde.java

```diff
@@ -91,7 +92,17 @@ public InputRow get()
                 true
             )
         );
-        agg.aggregate();
+        try {
+          agg.aggregate();
+        }
+        catch (ParseException e) {
+          // "aggregate" can throw ParseExceptions if a selector expects something but gets something else.
+          if (reportParseExceptions) {
```

Comment:
isn't this backwards? should be `reportParseExceptions`?

if so needs unit test